alerting-config-service-impl/src/main/java/org/hypertrace/alerting/config/service/EventConditionConfigServiceRequestValidator.java

```diff
@@ -0,0 +1,46 @@
+package org.hypertrace.alerting.config.service;
+
+import com.google.common.base.Preconditions;
+import org.hypertrace.alerting.config.service.v1.CreateEventConditionRequest;
+import org.hypertrace.alerting.config.service.v1.DeleteEventConditionRequest;
+import org.hypertrace.alerting.config.service.v1.GetAllEventConditionsRequest;
+import org.hypertrace.alerting.config.service.v1.UpdateEventConditionRequest;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+
+public class EventConditionConfigServiceRequestValidator {
+
+  public void validateCreateEventConditionRequest(
+      RequestContext requestContext, CreateEventConditionRequest request) {
+    validateTenantID(requestContext);
+    Preconditions.checkArgument(request.hasNewEventCondition(), "EventCondition should be present");
+    Preconditions.checkArgument(
+        request.getNewEventCondition().getMetricAnomalyEventCondtion() != null,
```

Comment:
proto messages will never return null, this should be a has check (or use the common validation utils).

Reply:
done